tutorials/using-bucket-website-with-mkdocs/index.mdx

```diff
@@ -0,0 +1,380 @@
+          font:
+            text: Roboto
+        ```
+
+        Refer to the [documentation](https://squidfunk.github.io/mkdocs-material/setup/changing-the-fonts/#changing-the-fonts) to further configure your website's font styles.
+
+        Your `mkdocs.yml` file should look like the following:
+
+          ```
+            site_name: My documentation website
+            site_url: https://mydomain.org/mysite
+            theme:
+              name: material
+              palette:
+                # Palette toggle for light mode
+                - scheme: default
+                  primary: black
+                  # Primary defines the primary color of the theme.
+                  accent: purple
+                  # Accent specifies the accent color of the theme.
+                  toggle:
+                  # toggle specifies the icon and name for toggling between light and dark modes.
+                    icon: material/toggle-switch
+                    name: Switch to dark mode
+
+                # Palette toggle for dark mode
+                - scheme: slate
+                  primary: white
+                  # primary defines the primary color of the theme.
+                  accent: deep purple
+                  # accent specifies the accent color of the theme.
+                  toggle:
+                  # toggle specifies the icon and name for toggling between light and dark modes.
+                    icon: material/toggle-switch-off-outline
+                    name: Switch to light mode
+              font:
+                text: Roboto
+          ```
+
+    ### Configuring navigation
+
+        MkDocs provides multiple ways to configure your website's navigation depending on the amount of content your website contains. Find out [all available configurations for navigation](https://squidfunk.github.io/mkdocs-material/setup/setting-up-navigation/) on the MkDocs website.
+
+        Set up the navigation structure of your website using the code sample below. This configuration allows you to organize your navigation menu as you wish.
+
+        ```
+        nav:
+        # Defines the navigation structure of the website, linking to the corresponding markdown files.
+          - Welcome to my documentation website: index.md
+          - My documentation website's FAQ: faq.md
+          - Understanding a concept in my documentation website: understanding-concept.md
+          - How to perform action one: perform-action-one.md
+          - How to perform action two: perform-action-two.md
+          - How to perform action three: perform-action-three.md
+        ```
+
+    ### Configuring admonitions
+
+        You can use admonitions to add side content to your documentation or make parts you want your users to notice stand out more. Refer to the [documentation](https://squidfunk.github.io/mkdocs-material/reference/admonitions/?h=admonitions#admonition-icons) to see available admonition icons.
+
+    #### Enabling admonitions
+
```

Comment:
Maybe you could explain what admonitions are briefly? (I don't think I've ever heard this)